WORKSPACE

```diff
@@ -602,3 +602,12 @@ custom_exec_properties(
         "BESPOKE_NETWORK_ON": create_exec_properties_dict(docker_network = "standard"),
     },
 )
+
+load("@exec_properties//:constants.bzl", "DOCKER_PRIVILEGED")
+
+# Same as @rbe_default_exec_properties execept that the exec properties also include DOCKER_PRIVILEGED.
```

Comment:
I was noticing all the examples include turning on a single exec property, how will this work if I need DOCKER_PRIVILEGED and NETWORK_ON?

Reply:
load("//rules/experimental/rbe:exec_properties.bzl", "merge_dicts")